### PR TITLE
Update `LobstersCard` UI

### PR DESCRIPTION
**Pull request by msasikanth**
Also updated top app bar scroll behaviour, since it was a small change merged it into this PR. Let me know if this needs to be separated out

### CODE REVIEW

File: common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt [hunk 1]
```diff
@@ -46,32 +51,39 @@ fun LobstersCard(
...
         post = post,
       )
-      Row(
-        modifier = Modifier.fillMaxWidth().padding(bottom = 8.dp),
-        verticalAlignment = Alignment.CenterVertically,
-        horizontalArrangement = Arrangement.End,
+      Column(
+        modifier = Modifier.weight(0.15f).fillMaxHeight(),
+        verticalArrangement = Arrangement.spacedBy(8.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
```
Comment (msfjarvis) [inline common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt:66]:
```suggestion
        horizontalAlignment = Alignment.CenterHorizontally,
```

File: common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt [hunk 2]
```diff
@@ -46,32 +51,39 @@ fun LobstersCard(
...
       ) {
         SaveButton(
           isSaved = localSavedState,
           modifier =
-            Modifier.clickable {
+            Modifier.clickable(
+              role = Role.Button,
+              indication = rememberRipple(bounded = false, radius = 24.dp),
+              interactionSource = remember { MutableInteractionSource() }
```
Comment (msfjarvis) [inline common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt:74]:
```suggestion
              interactionSource = remember { MutableInteractionSource() },
```

File: common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt [hunk 3]
```diff
@@ -122,40 +130,16 @@ fun SubmitterName(
   Row(
     modifier = modifier,
     verticalAlignment = Alignment.CenterVertically,
+    horizontalArrangement = Arrangement.spacedBy(4.dp)
```
Comment (msfjarvis) [inline common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt:133]:
```suggestion
    horizontalArrangement = Arrangement.spacedBy(4.dp),
```

File: android/src/main/kotlin/dev/msfjarvis/claw/android/ui/LobstersApp.kt [hunk 4]
```diff
@@ -98,10 +101,12 @@ fun LobstersApp(
       }
 
       Scaffold(
+        modifier = Modifier.nestedScroll(scrollBehavior.nestedScrollConnection),
         topBar = {
           ClawAppBar(
-            backgroundColor = systemBarsColor,
             modifier = Modifier.statusBarsPadding(),
+            backgroundColor = systemBarsColor,
+            scrollBehavior = scrollBehavior
```
Comment (msfjarvis) [inline android/src/main/kotlin/dev/msfjarvis/claw/android/ui/LobstersApp.kt:109]:
```suggestion
            scrollBehavior = scrollBehavior,
```

File: common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt [hunk 5]
```diff
@@ -46,32 +51,39 @@ fun LobstersCard(
 ) {
   var localSavedState by remember(post, isSaved) { mutableStateOf(isSaved) }
   Box(modifier = modifier.clickable { postActions.viewPost(post.url, post.commentsUrl) }) {
-    Column(
-      modifier = Modifier.padding(horizontal = 16.dp, vertical = 4.dp).fillMaxWidth(),
-      verticalArrangement = Arrangement.spacedBy(4.dp),
+    Row(
+      modifier = Modifier.fillMaxWidth().padding(16.dp),
+      horizontalArrangement = Arrangement.spacedBy(8.dp),
+      verticalAlignment = Alignment.CenterVertically
```
Comment (msfjarvis) [inline common/src/commonMain/kotlin/dev/msfjarvis/claw/common/posts/LobstersCard.kt:57]:
```suggestion
      verticalAlignment = Alignment.CenterVertically,
```

**Review by msfjarvis — COMMENTED** [review]
just a bunch of missing trailing commas

**Comment by msasikanth** [comment]
> just a bunch of missing trailing commas

Is there an IDE setting to place them automatically? I don't usually use trailing commas.

**Comment by msfjarvis** [comment]
> > just a bunch of missing trailing commas
> 
> Is there an IDE setting to place them automatically? I don't usually use trailing commas.

Not to my knowledge, I tend to add them by hand.

**Pull request merged**